homeassistant/components/point/__init__.py

```diff
@@ -70,62 +78,96 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     conf = config[DOMAIN]
 
-    config_flow.register_flow_implementation(
-        hass, DOMAIN, conf[CONF_CLIENT_ID], conf[CONF_CLIENT_SECRET]
+    await async_import_client_credential(
+        hass,
+        DOMAIN,
+        ClientCredential(
+            conf[CONF_CLIENT_ID],
+            conf[CONF_CLIENT_SECRET],
+        ),
+    )
+
+    async_create_issue(
+        hass,
+        HOMEASSISTANT_DOMAIN,
+        f"deprecated_yaml_{DOMAIN}",
+        breaks_in_ha_version="2025.4.0",
+        is_fixable=False,
+        issue_domain=DOMAIN,
+        severity=IssueSeverity.WARNING,
+        translation_key="deprecated_yaml",
+        translation_placeholders={
+            "domain": DOMAIN,
+            "integration_title": "Point",
+        },
     )
 
     hass.async_create_task(
         hass.config_entries.flow.async_init(
-            DOMAIN, context={"source": config_entries.SOURCE_IMPORT}
+            DOMAIN, context={"source": SOURCE_IMPORT}, data=conf
         )
     )
```

Comment:
So why do we import again?

Reply:
Oh this is from YAML, this might be the good way since we don't have an entry yet